components/brave_wallet_ui/components/desktop/popup-modals/transaction_details_modal/transaction_details_modal.tsx

```diff
@@ -181,6 +184,16 @@ export const TransactionDetailsModal = ({ onClose, transaction }: Props) => {
     chainId: transaction.chainId,
     coin: txCoinType
   })
+
+  const { data: toNetwork } = useGetNetworkQuery(
+    isBridgeTx && transaction.swapInfo?.toChainId && transaction.swapInfo.toCoin
```

Comment:
`swapInfo.toCoin` could possibly be `0` which is a valid option but is "falsey"

Reply:
Good catch! Now checking for `undefined`.